### PR TITLE
feat(how-to guides): camera intrinsic calibration tool from Autocore

### PR DESCRIPTION
**Pull request by JianKangEgon**
## Description

Add a new tool for camera intrinsic calibration from Autocore.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by JianKangEgon** [comment]
@kenji-miyake  could you plz check this pr? many thanks

File: docs/how-to-guides/calibrating-your-sensors.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ Autoware expects to have multiple sensors attached to the vehicle as input to pe
 
 ## Camera calibration
 
-Navigation2 provides a [good tutorial for camera internal calibration](https://navigation.ros.org/tutorials/docs/camera_calibration.html).
+### Intrinsic Calibration
+
+- Navigation2 provides a [good tutorial for camera internal calibration](https://navigation.ros.org/tutorials/docs/camera_calibration.html).
+- Autocore provides a [light-weight tool](https://github.com/autocore-ai/calibration_tools/tree/main/camera_intrinsic_calib).
```
Comment (kenji-miyake) [inline docs/how-to-guides/calibrating-your-sensors.md:12]: @JianKangEgon Is it okay to use `Autocore`, not `AutoCore`?

**Comment by kenji-miyake** [comment]
@JianKangEgon Sure, could you read the guidelines and check the boxes?
image

**Comment by kenji-miyake** [comment]
And I've updated the rule of Conventional Commits recently, can I change the type from `docs` to `feat`?
https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/#example_2

File: docs/how-to-guides/calibrating-your-sensors.md (hunk 1 above)
Comment (JianKangEgon) [reply docs/how-to-guides/calibrating-your-sensors.md:12]: Autocore is OKay

**Comment by JianKangEgon** [comment]
> And I've updated the rule of Conventional Commits recently, can I change the type from `docs` to `feat`? https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/#example_2

yeah, of course

File: docs/how-to-guides/calibrating-your-sensors.md (hunk 1 above)
Comment (kenji-miyake) [reply docs/how-to-guides/calibrating-your-sensors.md:12]: Which is correct? I believe it's better to be consistent.

Comment (JianKangEgon) [reply docs/how-to-guides/calibrating-your-sensors.md:12]: @kenji-miyake sorry for the late reply, last week was our national holiday. I've double checked, to be consistent, AutoCore would be better, I'll close this pr and propose a new one @Sharrrrk,  many thanks

**Comment by JianKangEgon** [comment]
replace Autocore with AutoCore

**Comment by kenji-miyake** [comment]
@JianKangEgon Oh, you didn't need to create a new pull request, but it's okay. Thank you.
https://github.com/autowarefoundation/autoware-documentation/pull/208

**Pull request closed without merging**